README.md

```diff
@@ -61,73 +58,6 @@ Let's see, if I can do so?! I am new to Lua and modding, but someone said to me:
 
 Installation, configuration and support can be found on the [docs](https://ismoh.github.io/NoitaMP/).
 
-## Our awesome and valuable Contributors
-
-<a href="https://github.com/Ismoh/NoitaMP/graphs/contributors">
-  <img src="https://contrib.rocks/image?repo=Ismoh/NoitaMP" />
-</a>
-
-## Credits, Supporter and special thanks
-
-I wouldn't be able to create this mod without the help by
-
-- [@EvaisaGiac](https://github.com/EvaisaGiac/)
-- [@TheHorscht](https://github.com/TheHorscht/)
-- [@Pyry](https://github.com/probable-basilisk)
-- [@DevonX](https://github.com/DevonX)
-- [@shebpamm](https://github.com/shebpamm)
-- @Coxas/Thighs
-- [@dextercd](https://github.com/dextercd)
-- [@BlueAmulet](https://github.com/BlueAmulet)
-- [@Shaw](https://github.com/ShawSumma)
-- [@Ramiels](https://github.com/Ramiels)
-- [@bruham](https://steamcommunity.com/id/bruham/myworkshopfiles/?appid=881100)
-- [@ofoxsmith](https://github.com/ofoxsmith)
-- [@Myzumi](https://github.com/Myzumi)
-
-I also want to say thank you to the sponsors of this project on
```

Comment:
A reference (link) on README.md to this section (sponsor links) in mkdocs would be good. Can you add it?